src/App.js

```diff
@@ -0,0 +1,73 @@
+import "./App.css";
+import MainLayout from "./layouts/MainLayout/MainLayout";
+import {
+  Navigate,
+  RouterProvider,
+  createBrowserRouter,
+} from "react-router-dom";
+import TasksPage from "./pages/TasksPage/TasksPage";
+import UsersPage from "./pages/UsersPage/UsersPage";
+import UserPage from "./pages/UserPage/UserPage";
+import { QueryClientProvider } from "@tanstack/react-query";
+import { queryClient } from "./utils/vars";
+import AuthenticationPage from "./pages/AuthenticationPage/AuthenticationPage";
+import { checkSession, getToken, isAuthorized } from "./utils/utils";
+import { redirect } from "react-router-dom";
+import ErrorPage from "./pages/ErrorPages/ErrorPage";
+
+function App() {
+  //
+  const router = createBrowserRouter([
+    {
+      path: "/",
+      element: <MainLayout />,
+      errorElement: <ErrorPage />,
+      children: [
+        {
+          path: "/",
+          element: <Navigate to={"/auth"} />,
```

Comment:
user should not be redirected to auth page

Reply:
at `"/auth"` path I verify the user then user can access the page!